### PR TITLE
Fix typos preventing smart contract compilation.

### PR DESCRIPTION
**Pull request by eolszewski**

### CODE REVIEW

File: contracts/GuildBank.sol [hunk 1]
```diff
@@ -56,21 +56,21 @@ contract GuildBank is Ownable {
...
-        safeReedemId = safeRedeemId.add(1);
+        safeRedeemId = safeRedeemId.add(1);
 
         uint256 totalLootTokens = lootToken.totalSupply();
 
         require(lootToken.transferFrom(msg.sender, this, lootAmount), "GuildBank::redeemLootTokens - lootToken transfer failed");
 
         // burn lootTokens - will fail if approved lootToken balance is lower than lootAmount
-        lootToken.burn(msg.sender, lootAmount);
+        lootToken.proxyBurn(msg.sender, lootAmount);
```
Comment (ameensol) [inline contracts/GuildBank.sol]: I think this should be a burn, and it should be the GuildBank contract address passed in, not the msg.sender.

Comment (eolszewski) [reply contracts/GuildBank.sol]: Updated.

**Comment by rhlsthrm** [comment]
Going to merge this, as the current contract does not compile.

**Comment by eolszewski** [comment]
Yee

**Pull request merged**